python/mscclpp/language/tests/test.json

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
Maybe you could try `FracturedJson` in vscode extensions. Will make json more readable